cmake/GenerateVersion.cmake

```diff
@@ -7,19 +7,8 @@ set(NVIM_VERSION_MEDIUM
     "v${NVIM_VERSION_MAJOR}.${NVIM_VERSION_MINOR}.${NVIM_VERSION_PATCH}${NVIM_VERSION_PRERELEASE}")
 
 execute_process(
-  COMMAND git describe --first-parent --dirty
-  OUTPUT_VARIABLE GIT_TAG
-  ERROR_VARIABLE ERR
-  RESULT_VARIABLE RES
-)
-
-if(NOT RES EQUAL 0)
```

Comment:
The command can still fail if you're not building from a git clone (e.g., release tarball).

Reply:
Hmm, good catch.

Cmake won't fail even if `git describe` fails so I think it should be fine. I've modified the code to default to `vx.y.z-dev` if `git describe` fails.

Reply:
Fixed. Please take a look again.